lib/hexpm/web/templates/docs/publish.html.md

```diff
@@ -0,0 +1,154 @@
+```
+
+Once this step has been completed, check your email inbox for the confirmation email. Once you have followed the enclosed link, your account will be ready to use.
+
+### Naming your package
+
+Before publishing, you will have to choose the name of your package. Remember that packages published to Hex are public and can be accessed by anyone in the community. It is also the responsibility of the community to pick and encourage good package names. Here are some tips:
+
+  * Avoid using offensive or harassing package names, nicknames, or other identifiers that might detract from a friendly, safe, and welcoming environment for all.
+  * If you are providing functionality on top of an existing package, consider using that package name as a prefix. For example, if you want to add authentication to [Plug](https://github.com/elixir-lang/plug), consider calling your package `plug_auth` (or `plug_somename`) instead of `auth` (or `somename`).
+  * Avoid namespace conflicts with existing packages. Plug owns the `Plug` namespace, if you have an authentication package for Plug use the namespace `PlugAuth` instead of `Plug.Auth`.
+
+With a name in hand, it is time to add the proper metadata to your `mix.exs` file.
+
+### Adding metadata to `mix.exs`
+
+The package is configured in the `project` function in the project's `mix.exs` file. [See below](#example-mix-exs-file) for an example file.
+
+First, make sure that the `:version` property is correct. All Hex packages are required to follow [semantic versioning](http://semver.org/).
+
+Fill in the `:description` property. It should be a sentence, or a few sentences, describing the package.
+
+Under the `:package` property are some additional configuration options:
+
+<dl class="dl-horizontal">
+  <dt><code>:licenses</code></dt>
+  <dd>A list of licences the project is licensed under. This attribute is required.</dd>
+  <dt><code>:name</code></dt>
+  <dd>The name of the package in case you want to publish the package with a different name than the application name.</dd>
+  <dt><code>:maintainers</code></dt>
+  <dd>A list of names (and/or emails) of maintainers to the project. Optional but highly recommended.</dd>
+  <dt><code>:links</code></dt>
+  <dd>A map where the key is a link name and the value is the link URL. Optional but highly recommended.</dd>
+  <dt><code>:files</code></dt>
+  <dd>A list of files and directories to include in the package. Has to include <code>mix.exs</code>. Defaults to standard project directories, so you usually don't need to set this property.</dd>
+  <dt><code>:build_tools</code></dt>
+  <dd>List of build tools that can build the package. It's very rare that you need to set this, as Hex tries to automatically detect the build tools based on the files in the package. If a <code>rebar</code> or <code>rebar.config</code> file is present Hex will mark it as able to build with rebar. This detection can be overridden by setting this field.</dd>
+</dl>
```

Comment:
For some reason, earmark would completely skip any markdown rendering inside HTML, so the tags here (e.g. `code`) had to be preserved.